crates/matrix-sdk-crypto/src/backups/mod.rs

```diff
@@ -64,20 +64,10 @@ pub struct BackupMachine {
 struct PendingBackup {
     request_id: OwnedTransactionId,
     request: KeysBackupRequest,
+    /// Room ID : Sender Key : [Session IDs]
```

Comment:
If you believe this is necessary and can improve code readibility, we can add private type aliases:

```rust
type SenderKey = String;
type SessionId = String;
```

Reply:
Thanks, done in https://github.com/matrix-org/matrix-rust-sdk/pull/2951